Added new login page named login-form-v18

tamojeetK opened the pull request:
✅Saved my changes by following the instructions.
✅Uploaded the screenshot of my page.
✅Added profile details in Awesome Login pages README.md file (Name | Desktop Screenshot | Contributed by).

Please let me know if any further changes are needed to be done.

tamojeetK committed:
Create README.md

tamojeetK committed:
Added files

tamojeetK committed:
Create style.css

tamojeetK committed:
Added images

tamojeetK committed:
Added images

tamojeetK committed:
Update ReadMe.md

tamojeetK committed:
Update ReadMe.md

tamojeetK committed:
Add files via upload

tamojeetK committed:
U

abhir9 commented:
@tamojeetK 

1. another PR merged for folder 17 , please update the folder name
2. rename the screenshot name to "screenshot".extention
3. please update the mobile screenshot as well

tamojeetK committed:
Rename login-form-v17/README.md to login-form-v18/README.md

tamojeetK committed:
Create index.html

tamojeetK committed:
Create style.css

tamojeetK committed:
Create README.md

tamojeetK committed:
Add files via upload

tamojeetK committed:
Update README.md

tamojeetK committed:
Add files via upload

tamojeetK committed:
Transferred to login-form-v18

tamojeetK committed:
Update README.md

tamojeetK commented:
> @tamojeetK
> 
> 1. another PR merged for folder 17 , please update the folder name
> 2. rename the screenshot name to "screenshot".extention
> 3. please update the mobile screenshot as well

@abhir9 Sir I have updated the Folder and renamed it. Please check it
